docs/application/dotnet/guides/applications/service_application.md

```diff
@@ -0,0 +1,194 @@
+# Service Applications 
+
+Service applications are Tizen .NET applications with no graphical user interface that run in the background. They can be very useful in performing activities (such as getting sensor data in the background) that need to run periodically or continuously, but do not require any user intervention.
+
+The service application type allows creating reusable and independent parts which are really important in bigger projects where we can easily split responsibilities of the application into different parts. 
+Consider the speedometer application designed for average speed measurement; the service part, in this case, is responsible for reading speed from the device GPS module, it also calculates the average speed. The UI communicates with the service when required and shows the measured values to the user. 
+This approach allows reusing modules between different applications such as cyclist applications, automotive solutions, and sport activity apps. The core module, with business logic, works in the background and performs the task it was designed for until it gets closed by system events like `OnLowMemory` or `OnLowBattery`. The activity of the core module remains unaffected by any user action such as accidental closing of the app UI.
+
+The main service application API features include: 
```

Comment:
API names are common compound nouns. So capitalized one is correct.
Ref: https://github.com/Samsung/tizen-docs/blob/master/styleguide/naming-rules.md#proper-nouns
```suggestion
The main Service Application API features include: 
```